web/html/src/manager/admin/list-payg/list-payg.tsx

```diff
@@ -0,0 +1,200 @@
+import { hot } from "react-hot-loader/root";
+import * as React from "react";
+import { useEffect } from "react";
+import _truncate from "lodash/truncate";
+import { Column } from "components/table/Column";
+import { SearchField } from "components/table/SearchField";
+import { Table } from "components/table/Table";
+import { Utils } from "utils/functions";
+import { Button } from "components/buttons";
+import { showSuccessToastr } from "components/toastr/toastr";
+import withPageWrapper from "components/general/with-page-wrapper";
+import { ServerMessageType } from "components/messages";
+import { FromNow } from "components/datetime/FromNow";
+import { HelpLink } from "components/utils";
+import { SectionToolbar } from "components/section-toolbar/section-toolbar";
+import PaygStatus from "manager/admin/payg-shared/common/payg-status";
+
+const _SETUP_WIZARD_STEPS = [
```

Comment:
It'd be great to have a few screenshots in the PR description.

Reply:
The PR description already has some screenshots. 
However, I just notice that they are not up-to-date. I will update them.
There is any specific page that you this is missing?